### ISSUE TITLE
Disable in certain apps [enhancement]

### ISSUE THREAD

**Issue by papertree0** [noah-nuebling/Mac-Mouse-Fix#933]
## Description
You should have the ability to turn off Mac Mouse Fix in certain apps. 

## Rationale
There are apps where the controls (namely the "Keyboard Modifiers" in the Scrolling section collide with other functions and cause problems. 
For example, on Ableton Live, Cmd Wheel zooms in. With MMF active (and having set up a modifier to do that same thing with Cmd Wheel) the actions collide and nothing happens.

## Additional info
–

**Comment by belafonte** [comment]
Also many 3D Programs use the middle mouse button, which is not working anymore.

**Comment by TheRoyalScam** [comment]
I actually came here to request the exact same feature. MOS has this feature, where we can whitelist any app, and MOS is disabled only in that app. 

I prefer your app’s smooth scrolling over MOS, but Avid Pro Tools goes wonky with any smooth scrolling enabled. If you added this feature, I’d come back to MMF in a heartbeat.

**Comment by Shavcat** [comment]
Yeah, It's definately would be useful. Right now I'm trying to find workaround with Cinema 4d where middle button is critical

**Comment by ghost** [comment]
强烈建议增加应用程序排除功能，强烈建议，强烈建议。

**Comment by testfailed** [comment]
It is my most needed feature. (And I guess it would be the most needed feature for any user too.)

And also it would be great if there's an additional modifier key (^ key or customizable, just like the other ones) to temporarily disable smooth scrolling while pressing it.

Thanks for the great app.
And I want to keep using it.

**Comment by rmtmckenzie** [comment]
Blender is impossible to use with smoothing set to high. This would fix that.

**Comment by chuyskywalker** [comment]
Fusion is also quite frustrating to use with scroll smoothing turned on; having to turn it on/off manually is quite the ritual annoyance.
